TACC/Core-Portal
# Task/FP-195: Fix django impersonate feature

## Overview: ##
Makes impersonate work.

## Related Jira tickets: ##

* [FP-195](https://jira.tacc.utexas.edu/browse/FP-195)

## Summary of Changes: ##
Fixes url and add impersonate account list item to make impersonate work properly.

## Testing Steps: ##
1. Login as a superuser.
2. Find `user.id` of a non-superuser test user you want to impersonate (Maybe wtih `get_user_model()` ).
3. Go to the impersonate link with that id (`https://cep.dev/core/admin/impersonate/<userId>`).
4. Ensure you see the website as the test user sees it.
5. Ensure that user account drop-down has the `Stop impersonating` item.

## UI Photos:
![Selection_008](https://user-images.githubusercontent.com/9425579/128383302-7a7d53d1-3ca7-45d5-b073-b870ad21b31f.png)

## Notes: ##
